public/locales/es/translation.json

```diff
@@ -0,0 +1,193 @@
+      "emergencies": "Emergencias",
+      "emotional_attraction": "Atracción emocional",
+      "emotional_exclusivity": "Exclusividad emocional",
+      "empathy": "Empatía",
+      "end_of_life_death": "Fin de la vida / Muerte",
+      "events": "Eventos",
+      "writing": "Escritura",
+      "executor_of_will": "Testamentarie",
+      "family": "Familia",
+      "family_sick_or_elderly": "Familia (enfermedad y vejez)",
+      "fetishes": "Fetiches",
+      "friend": "Amigue",
+      "friends": "Amigues",
+      "friendship": "Amistad",
+      "gaming": "Juegos",
+      "genitals": "Genitales",
+      "hand_holding": "Tomarse de la mano",
+      "health": "Salud",
+      "holiday": "Ir de vacaciones",
+      "hugs": "Abrazos",
+      "improv": "Improvisación",
+      "intellectual_discussions": "Discusiones intelectuales",
+      "kisses": "Besos",
+      "kissing": "Besarse",
+      "listening": "Escuchar",
+      "love_languages": "Lenguajes de amor",
+      "m_s": "M/s",
+      "marriage": "Matrimonio",
+      "martial_arts": "Artes marciales",
+      "massage": "Masaje",
+      "mentor_guide": "Mentore - Guía",
+      "music": "Música",
+      "nudity": "Desnudez",
+      "organization": "Organización",
+      "orgasms": "Orgasmos",
+      "partner": "Compañere",
+      "payments": "Pagos",
+      "plants": "Plantas",
+      "power_of_attorney": "Poder notarial",
+      "projects": "Proyectos",
+      "property": "Propiedad",
+      "reliability": "Fiabilidad",
+      "responsibilities": "Responsabilidades",
+      "routine": "Rutina",
+      "routines": "Rutinas",
+      "sadomasochism": "Sadomasoquismo",
+      "seen_together": "Vistos juntes",
+      "sexual_exclusivity": "Exclusividad sexual",
+      "shared_accounts": "Cuentas compartidas",
+      "shared_activities": "Actividades compartidas",
+      "shared_beliefs": "Creencias compartidas",
+      "shared_experiences": "Experiencias compartidas",
+      "shared_goals": "Metas compartidas",
+      "shared_interests": "Intereses compartidos",
+      "shared_ritual": "Ritual compartido",
+      "shared_values": "Valores compartidos",
+      "sharing_a_home": "Compartir un hogar",
+      "sharing_a_sleeping_space": "Compartir un espacio para dormir",
+      "sharing_beliefs": "Compartir creencias",
+      "sharing_meals": "Compartir comidas",
+      "platonic": "Platónico",
+      "comet": "Cometa",
+      "long_distance": "A larga distancia",
+      "sharing_values": "Compartir valores",
+      "shibari": "Shibari",
+      "social_media": "Redes sociales",
+      "sponsor_sponsee": "Patrocinadore - Padrine",
+      "spouse": "Cónyuge",
+      "support": "Apoyo",
+      "teacher_student": "Profesoree - Estudiante",
+      "teaching": "Enseñanza",
+      "theatre": "Teatro",
+      "visual_art": "Arte visual",
+      "work": "Trabajo",
+      "working_out": "Ejercicio",
+      "yoga": "Yoga",
+      "daily": "Diario",
+      "weekly": "Unas veces a la semana",
```

Comment:
@duizendnegen does "weekly" here read as "once a week" or "a few times a week"?

Reply:
Once a week

Reply:
So it would be "Una vez a la semana" in singular